service/code_interpreter.py

```diff
@@ -73,36 +82,65 @@ async def __aexit__(self, _exc_type, _exc_value, _traceback):
             self.sandbox.keep_alive(self.timeout)
         self.sandbox.close()
 
-    def get_files_code(self):
+    def get_dataframe(self):
         """
         Get the code to read the files in the sandbox.
         This can be used for instructing the LLM how to access the loaded files.
         """
-
         # TODO: Add support for xslx, json
```

Comment:
Right now this works only for one file, right?
```suggestion
        # TODO: Add support for multiple dataframes
```